src/main/resources/assets/jei/lang/pt_br.json

```diff
@@ -2,60 +2,60 @@
   "_comment": "Tooltips",
   "jei.tooltip.config": "Configurações do JEI",
   "jei.tooltip.show.recipes": "Mostrar receitas",
-  "jei.tooltip.show.all.recipes": "Mostrar todas as receitas",
+  "jei.tooltip.show.all.recipes": "Todas as receitas",
   "jei.tooltip.delete.item": "Clique para excluir",
-  "jei.tooltip.liquid.amount.with.capacity": "%s / %s mB",
-  "jei.tooltip.liquid.amount": "%s mB",
+  "jei.tooltip.liquid.amount.with.capacity": "%,d / %,d mB",
+  "jei.tooltip.liquid.amount": "%,d mB",
```

Comment:
Is there a reason why you have backed out changes from commit fc2e33694fa9be82c488e794f278aba0e2874031? I suppose it was unintended ;)

Reply:
yeah, thanks for pointing that out

Reply:
@DaDodger this has not been resolved yet, maybe you didn't push your commit?

Reply:
@mezz I guess it's all good now. Ready to merge.